build/msys2-geany-plugins-release.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+
+import os
+import shutil
+import sys
+import glob
+from subprocess import check_call
+from os.path import exists, isfile, join
+
+"""
+This script prepares a Geany release on Windows.
+The following steps will be executed:
+- strip binary files (geany.exe, plugin .dlls)
+- sign binary files with certificate
+- create installers
+- sign installers
+"""
+
+VERSION = '2.1'
+# adjust paths to your needs ($HOME is used because expanduser() returns the Windows home directory)
+BASE_DIR = join(os.environ['HOME'], 'geany_build')
+SOURCE_DIR = join(os.environ['HOME'], 'git', 'geany-plugins')
+RELEASE_DIR_ORIG = join(BASE_DIR, 'release', 'geany-plugins-orig')
+RELEASE_DIR = join(BASE_DIR, 'release', 'geany-plugins')
+BUNDLE_BASE_DIR = join(BASE_DIR, 'bundle')
+BUNDLE_GEANY_PLUGINS = join(BASE_DIR, 'bundle', 'geany-plugins-dependencies')
+INSTALLER_NAME = join(BASE_DIR, f'geany-plugins-{VERSION}_setup.exe')
+
+# signing params
```

Comment:
Similarly to what discussed in https://github.com/geany/geany/pull/4223 as @b4n questioned the signing part of the script, @eht16 then commented that he explained parts of the signing topic in https://github.com/geany/geany/issues/4189#issuecomment-2629001461 and we will probably not sign the binaries any longer and so we could also remove the related code.